### PR TITLE
Add annotation changes to version Istio 1.6.8

### PR DESCRIPTION
**Pull request by brendarearden**
Rebuild Istio 1.6.8 with the newest set of annotation and chart changes

Updated to use cattle system registry

**Issue**
https://github.com/rancher/rancher/issues/28818

### CODE REVIEW

File: packages/rancher-istio/charts/configs/istio-base.yaml [hunk 1]
```diff
@@ -4,6 +4,14 @@ spec:
...
     istiocoredns:
       enabled: {{ .Values.istiocoredns.enabled }}
+    prometheus:
+      enabled: false
+    grafana:
+      enabled: false
+    kiali:
+      enabled: false
+    tracing:
+      enabled: false
```
Comment (cbron) [inline packages/rancher-istio/charts/configs/istio-base.yaml:14]: Why'd these come back ?

Comment (brendarearden) [reply packages/rancher-istio/charts/configs/istio-base.yaml:14]: This pr shows them getting removed only when going to 1.7 https://github.com/rancher/charts/pull/581/files. We had them in as they still had default values in 1.6 so we wanted to ensure they were always false.

File: packages/rancher-istio/charts/values.yaml [hunk 2]
```diff
@@ -54,13 +54,14 @@ sidecarInjectorWebhook:
...
   proxy:
     repository: rancher/istio-proxyv2
-    tag: 1.7.0
+    tag: 1.6.8
   proxy_init:
     repository: rancher/istio-proxyv2
-    tag: 1.7.0
+    tag: 1.6.8
   defaultPodDisruptionBudget:
     enabled: true
```
Comment (cbron) [inline packages/rancher-istio/charts/values.yaml:67]: Kiali stays at same version too now ?

Comment (brendarearden) [reply packages/rancher-istio/charts/values.yaml:67]: Yes, kiali version didn't change from 1.6.8 to 1.7.0. The new version was released later. I will be updating the kiali version in my next pr with some kiali chart changes as well (I still need to do the `global.cattle.systemDefaultRegistry` for kiali and a few other things)

**Review by paynejacob — APPROVED** [review]

File: packages/rancher-istio/charts/configs/istio-base.yaml (hunk 1 above)
Comment (cbron) [reply packages/rancher-istio/charts/configs/istio-base.yaml:14]: Guess it doesn't matter if we are deleting 1.6 anyway

File: packages/rancher-istio/charts/values.yaml (hunk 2 above)
Comment (cbron) [reply packages/rancher-istio/charts/values.yaml:67]: ok

**Review by cbron — APPROVED** [review]

**Review by rmweir — APPROVED** [review]
LGTM

**Pull request merged**